tests/snuba/api/endpoints/test_event_details.py

```diff
@@ -7,12 +7,15 @@
 from django.core.urlresolvers import reverse
 
 from sentry.models import UserReport, Group
-from sentry.testutils import APITestCase, SnubaTestCase
+from sentry.testutils import APITestCase, SnubaTestMixin
+from sentry.testutils.skips import requires_snuba
 
 
-class EventDetailsTest(APITestCase, SnubaTestCase):
+@requires_snuba
+class EventDetailsTest(SnubaTestMixin, APITestCase):
     def setUp(self):
         super(EventDetailsTest, self).setUp()
+        self.init_snuba()
```

Comment:
I don't really like that we need to call this for every test using the mixin. Could we keep `SnubaTestCase` and just have it call `init_snuba` for us? Not sure there are any cases in the existing tests where we don't want this to happen

Reply:
Its possible, I was bumping into issues with inheritance with `AcceptanceTestCase`. I can revisit that though.